bigtable/google/cloud/bigtable/table.py

```diff
@@ -346,6 +346,14 @@ def read_row(self, row_key, filter_=None):
         :param filter_: (Optional) The filter to apply to the contents of the
                         row. If unset, returns the entire row.
 
+        :type retry: :class:`~google.api_core.retry.Retry`
+        :param retry:
+            (Optional) Retry delay and deadline arguments. To override, the
+            default value :attr:`DEFAULT_RETRY_READ_ROWS` can be used and
+            modified with the :meth:`~google.api_core.retry.Retry.with_delay`
+            method or the :meth:`~google.api_core.retry.Retry.with_deadline`
+            method.
```

Comment:
This would be clearer with examples, e.g.:

```python
        :type retry: :class:`~google.api_core.retry.Retry`
        :param retry:
            (Optional) Retry policy. Override, the default via e.g.:
            ``DEFAULT_RETRY_READ_ROWS.with_deadline(new_value)`` or
            ``DEFAULT_RETRY_READ_ROWS.with_delay(new_value)``.
```

Reply:
Certainly. Thanks. Done.